_posts/2023-01-16-Introducing-Identity.markdown

```diff
@@ -0,0 +1,44 @@
+# Introducing Identity for OpenSearch
+
+OpenSearch's existing access control features included in the Security plugin let administrators apply access control to indexes and cluster actions so that users have the right permissions to do their work and the cluster is protected from unwanted activity. However, the features for access control that are currently in place do have certain limitations that can make it hard to use them with other plugins.
+
+As the core OpenSearch project begins its shift away from a plugin model to a framework that utilizes extensions, those extensions, its legacy plugins, and the administrators who manage them will need mechanisms for controlling access that are more granular and able to cover a broader range of scenarios where effective access control is critical. We are creating/building out/developing a new suite of features that are designed to provide comprehensive access control to OpenSearch’s ecosystem, and we collectively call these new features Identity. 
```

Comment:
wdyt about linking to a blog post or this GH issue (https://github.com/opensearch-project/OpenSearch/issues/2447) on extensions here?

Reply:
I've added a link when I first mention extensions